### ISSUE TITLE
Target Leakage

### ISSUE THREAD

**Issue by XY-ATOE** [idstcv/SeCu#6]
During the training process, it can be observed that the true labels of samples are also involved in constructing the training loss. Could this potentially lead to target leakage?

**Comment by XY-ATOE** [comment]
Alternatively, if I have a completely unlabeled dataset consisting of 10 classes, and I am unable to annotate the exact class for each image, how can I use code to train a model?

**Comment by XY-ATOE** [comment]
Thanks a lot！

**Comment by qian-qi** [comment]
Thank you for your interest. Please note that we have rewritten the folder function in secu/folder.py and the target in train_loader is a unique id for each image rather than the ground-truth label. 
Given a set of unlabeled images in a folder, the code can be applied directly without annotations.

**Comment by qian-qi** [comment]
I will close the issue and you can reopen it if there are further questions.
